### DISCUSSION TITLE
Question: Name of chunkFiles

### DISCUSSION THREAD

**Discussion by Tylopilus** [webpack/webpack#13257] [General]
Hi,

after I have upgraded from v4 to v5 I can see that my chunfiles have been renamed and now containing their relative path.

e.g. Vendor-chunk-Name
* on v4: vendors~site.js
* on v5: vendors-node_modules_jquery_dist_jquery_js.js

Since I did not make any other configuration my assumption is that this is due to the upgrade of webpack. My build-tools actually rely on the ~ in the name, so is there a way to revert to the old naming?

**Comment by alexander-akait** [comment]
Please use `Discussions` for questions, issues for bugs/features, you can set name for vendor chunk, what is the problem with new naming?
